src/Notes/ConfirmTaxSettings.php

```diff
@@ -33,7 +33,7 @@ public static function get_note() {
 		$note = new Note();
 
 		$note->set_title( __( 'Confirm tax settings', 'woocommerce-admin' ) );
-		$note->set_content( __( 'Automated tax calculations are enabled on your store through WooCommerce Shipping & Tax. Learn more about automated taxes <a href="https://docs.woocommerce.com/document/woocommerce-services/#section-12">here</a>.', 'woocommerce-admin' ) );
+		$note->set_content( __( 'Automated tax calculations are enabled on your store through WooCommerce Shipping & Tax. Learn more about automated taxes <a href="https://docs.woocommerce.com/document/woocommerce-services/#section-12?utm_medium=product">here</a>.', 'woocommerce-admin' ) );
```

Comment:
Same comment about `sprintf` here.

Reply:
Same question as above @joshuatf just checking if you're expecting the anchor tags in variables?

Reply:
@joshuatf it looks like this note is not actually used at the moment 😞 otherwise I would expect it to be called here https://github.com/woocommerce/woocommerce-admin/blob/425175708392918216cc8a76448f28ae429f75b4/src/Events.php?plain=1#L111

Reply:
ConfirmTaxSettings can be deleted. See the discussion: p1627526773416800-slack-C9K5T0XSP